test-container/pom.xml

```diff
@@ -23,6 +23,16 @@
             <groupId>org.testcontainers</groupId>
             <artifactId>testcontainers</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.github.spotbugs</groupId>
+            <artifactId>spotbugs-annotations</artifactId>
+        </dependency>
+        <!-- for TestUtils.waifFor       -->
+        <dependency>
+            <groupId>io.strimzi</groupId>
+            <artifactId>test</artifactId>
+            <scope>compile</scope>
+        </dependency>
```

Comment:
This will not work well once we move it to the sepsrate repo.

Reply:
I know but should I insert also the implementation of `waitFor` in some utils?